### DISCUSSION TITLE
Scenario window, "RESET" button

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#10303] [General]
I think it would be good to add a "RESET" button (or a "RESTORE" button) which would restore all the settings to their original states for the selected scenario, the default settings applied by the author of the scenario in the scenario editor.

![Scenario window 08](https://user-images.githubusercontent.com/43583994/90334197-3bee0b00-dfcc-11ea-9a23-8af827fcfd90.png)

**Comment by zenseii** [comment]
I think this could easily be achieved by just clicking SELECT and reloading the map?

This might've made more sense when it was possible to change the turn order of the players but that was removed when we added the handicap feature.

Should this be closed?

**Comment by ihhub** [comment]
Hi @LeHerosInconnu , please reply to @zenseii suggestion. Frankly speaking such a button would help literally in few situations. Most of players wouldn't use it.

**Comment by LeHerosInconnu** [comment]
Hello @zenseii and @ihhub,

> I think this could easily be achieved by just clicking SELECT and reloading the map?
> 
> This might've made more sense when it was possible to change the turn order of the players but that was removed when we added the handicap feature.
> 
> Should this be closed?

> Hi @LeHerosInconnu , please reply to @zenseii suggestion. Frankly speaking such a button would help literally in few situations. Most of players wouldn't use it.

Using the "SELECT" button would be neither very intuitive (how can you tell that the select button also has this function) nor very practical (back and forth manipulation).
A dedicated button visible directly in the interface would be more ergonomic.
I'd also suggest a "graphic" square button (not a button with text), which would avoid translations (current and future) and take up less space (more discreet).

Something like that:
Image for explanation only.

![Reset 50](https://github.com/ihhub/fheroes2/assets/43583994/f43ff155-1c7a-4341-a778-0134f755e868)

Furthermore, it should be a rule to add only graphic buttons, when possible, in the interface.

**Comment by ihhub** [comment]
Moving this issue to the Discussion topic as we haven't found an agreement and also we have no idea where the proposed new button could fit within the existing dialog.
